src/btree/bt_split.c

```diff
@@ -2163,6 +2163,10 @@ __split_multi_lock(WT_SESSION_IMPL *session, WT_REF *ref, int closing)
     WT_DECL_RET;
     WT_PAGE *parent;
 
+    /* Fail 1% of the time to simulate we fail to split the page. */
```

Comment:
As discussed, this was bumped to 1% instead of 0.1% as this branch is not as common as the previous location of the fail point.